control/controldisplay/formatter.go

```diff
@@ -52,7 +54,7 @@ type Formatter interface {
 func GetExportFormatter(exportFormat string) (Formatter, error) {
 	formatter, found := exportFormatters[exportFormat]
 	if !found {
-		return nil, fmt.Errorf("invalid export format '%s' - must be one of json,csv", exportFormat)
+		return nil, fmt.Errorf("invalid export format '%s' - must be one of json,csv,html", exportFormat)
```

Comment:
can we put valid formatters into a consta string array, then we can use it here and in `InferFormatFromExportFileName`